infra/ansible/playbooks/docker.yaml

```diff
@@ -44,4 +44,4 @@
         name: "{{ user }}"
         groups: docker
         append: yes
-        state: present
+        state: present
```

Comment:
Add back the newline at the end of the file, it's happening with all the telemetry `.service` as well